functions.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/** dbQuery function to fetch data from database
+ * @param string $query
```

Comment:
space between description and params, also between params and return
(please do for all docblocks)

Reply:
?